libs/block-editor/src/lib/extensions/components/suggestions/suggestions.component.ts

```diff
@@ -229,4 +245,18 @@ export class SuggestionsComponent implements OnInit, AfterViewInit {
                 this.resetKeyManager();
             });
     }
+
+    private sanitizeUrl(url: string): SafeUrl {
```

Comment:
use this one: https://github.com/dotCMS/core-web/blob/master/libs/block-editor/src/lib/utils/suggestion.utils.ts#L64